src/retabled/core.cljc

```diff
@@ -50,14 +50,21 @@
    :example/csv-download? "If truthy, a button will appear with the corresponding table for downloading the data in a csv file"
    :sort-filter-highlight "If truthy, columns will be highlighted when they are currently sorted or filtered. The default highlight color is light gray, but a string can be placed here as the truthy value to customize that color."})
 
+(defn add-unique-key
+  [items]
+  (let [assoc-gensym (fn [item] (assoc item :genkey (gensym)))]
```

Comment:
Let's use the 2-arg version of gensym here, attaching the `item` as the second arg so it is random (gensym) but readable in a stack trace (item)